tools/x-repo-types/xrt_test.go

```diff
@@ -63,6 +63,14 @@ func TestCrossRepoTypes(t *testing.T) {
 			skip:       true,
 			skipReason: `Several issues. For example: LEVEL 5 of goal bookkeeping.Block is EvalDelta with field [SharedAccts](codec:"sa,allocbound=config.MaxEvalDeltaAccounts") VS SDK types.EvalDelta is missing SharedAccts field`,
 		},
+		{
+			name:    "goal-v-sdk-eval-delta",
+			xPkg:    "github.com/algorand/go-algorand/data/transactions",
+			xType:   "EvalDelta",
+			yPkg:    "github.com/algorand/go-algorand-sdk/v2/types",
+			yBranch: "develop",
+			yType:   "EvalDelta",
+		},
```

Comment:
TODO: don't forget `ConsensusParams` in `config` vs. in go-algorand-sdk's `protocol.config`